.github/workflows/helm-chart.yaml

```diff
@@ -8,6 +8,18 @@ on:
       - "infra/charts/**"
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: The GitHub user or org that owns this repository
+        type: string
+        required: true
+      repository:
+        description:  The GitHub repository
+        type: string
+        required: true
+        default: "charts"
```

Comment:
To make it possible to manually trigger `publish-chart` pipeline from the dev branch